roswallpexpo.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = ReactOS Wallpaper Exposition
+Version = 5
+License = CC BY
+Description = A wallpaper set (apple, leaves, sky etc.), by fastcom.
+Category = 16
+URLSite = https://jira.reactos.org/browse/CORE-12639
+URLDownload = https://jira.reactos.org/secure/attachment/67625/RosWallpExpo-v5.exe
```

Comment:
I am not sure whether it would be wise to apply additional traffic onto our JIRA servers by adding such links to JIRA into rapps. I think the file should be placed on github for example. Which does offer servers with high availability, and stable links. And we don't have to pay/maintain that infrastructure ourself.

Reply:
We can use my private GitHub Release page or our official Release page for downloadinp. Official page is the better but I have no rights to modify our official Release page.

Reply:
Our release-page https://github.com/reactos/reactos/releases shouldn't get messed with "unrelated files" sprinkled in between our official ros-release-announcements.
Optimal for such stuff would be to create some project like https://github.com/reactos/hosted-files
As long as something like that doesn't exist (I do understand you don't have the rights for such a thing),
I do think **your private GitHub Release page is a sane alternative, yes**.
We do already have a lot of rapps links of the form https://github.com/someContributor/SomeProject